hub/api/tests/test_dataset_utils.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from hub.api.dataset_utils import _get_compressor
+
+CODECS = ("lz4", "zstd", "numpy", "png", "jpeg")
```

Comment:
compose this list from available classes in `hub.codecs.__init__` & use this same variable in `hub/exceptions.py`